# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/iptraf/path.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur iptraf-2.7.0.orig/CHANGES iptraf-2.7.0/CHANGES
--- iptraf-2.7.0.orig/CHANGES	2002-05-19 14:56:21.000000000 +0200
+++ iptraf-2.7.0/CHANGES	2003-06-02 23:03:14.000000000 +0200
@@ -657,8 +657,8 @@
 	Separated log file into different logs for each facility.
 
 	Moved log files to /var/log/iptraf to avoid mixing them with the
-	mess in the /var/local/iptraf directory.  At least that way,
-	we humans don't have to look in /var/local/iptraf anymore.
+	mess in the /var/iptraf directory.  At least that way,
+	we humans don't have to look in /var/iptraf anymore.
 
 	Relaxed multiple-instance restriction from a
 	no-multiple-instances-of-IPTraf requirement to a
diff -Nur iptraf-2.7.0.orig/Documentation/config.html iptraf-2.7.0/Documentation/config.html
--- iptraf-2.7.0.orig/Documentation/config.html	2002-05-19 14:16:31.000000000 +0200
+++ iptraf-2.7.0/Documentation/config.html	2003-06-02 23:01:01.000000000 +0200
@@ -82,7 +82,7 @@
   main menu. The configuration is stored in the
   <TT
 CLASS="FILENAME"
->/var/local/iptraf/iptraf.cfg</TT
+>/var/iptraf/iptraf.cfg</TT
 > file. If the file is not found, IPTraf uses
   the default settings. Any changes to the configuration immediately get
   stored in the configuration file.</P
@@ -495,4 +495,4 @@
 ></DIV
 ></BODY
 ></HTML
->
\ No newline at end of file
+>
diff -Nur iptraf-2.7.0.orig/Documentation/gettingstarted.html iptraf-2.7.0/Documentation/gettingstarted.html
--- iptraf-2.7.0.orig/Documentation/gettingstarted.html	2002-05-19 14:16:25.000000000 +0200
+++ iptraf-2.7.0/Documentation/gettingstarted.html	2003-06-02 23:01:51.000000000 +0200
@@ -513,7 +513,7 @@
 >    The resulting binaries will be placed in the
 <TT
 CLASS="FILENAME"
->/usr/local/bin</TT
+>/usr/bin</TT
 > directory.
     All needed directories will also be created.</P
 ><P
@@ -609,7 +609,7 @@
     ready-to-run precompiled software.  This will copy the binaries to
     <TT
 CLASS="FILENAME"
->/usr/local/bin</TT
+>/usr/bin</TT
 >, and
     create the necessary working directories.</P
 ></LI
@@ -649,7 +649,7 @@
 >  In both cases (downloaded and floppy), the installation will store the
   program in <TT
 CLASS="FILENAME"
->/usr/local/bin</TT
+>/usr/bin</TT
 > with the binaries owned by
   user root, readable, writable, and executable by the owner,
   no permissions for the group, no permissions for all others. (700 octal,
@@ -707,7 +707,7 @@
 ><P
 >  Be sure <TT
 CLASS="FILENAME"
->/usr/local/bin</TT
+>/usr/bin</TT
 > is included in
   your environment's <TT
 CLASS="ENVAR"
@@ -781,4 +781,4 @@
 ></DIV
 ></BODY
 ></HTML
->
\ No newline at end of file
+>
diff -Nur iptraf-2.7.0.orig/Documentation/iptraf.8 iptraf-2.7.0/Documentation/iptraf.8
--- iptraf-2.7.0.orig/Documentation/iptraf.8	2002-03-18 09:05:19.000000000 +0100
+++ iptraf-2.7.0/Documentation/iptraf.8	2003-06-02 23:02:30.000000000 +0200
@@ -86,7 +86,7 @@
 
 .SH FILES
  /var/log/iptraf/*.log - log file
- /var/local/iptraf/* - important IPTraf data files
+ /var/iptraf/* - important IPTraf data files
 
 .SH SEE ALSO
  Documentation/* - complete documentation written by the author
diff -Nur iptraf-2.7.0.orig/Documentation/messages.html iptraf-2.7.0/Documentation/messages.html
--- iptraf-2.7.0.orig/Documentation/messages.html	2002-05-19 14:16:32.000000000 +0200
+++ iptraf-2.7.0/Documentation/messages.html	2003-06-02 23:02:12.000000000 +0200
@@ -107,7 +107,7 @@
   program, and the necessary directory
 <TT
 CLASS="FILENAME"
->/var/local/iptraf</TT
+>/var/iptraf</TT
 > does not
   exist. Can also be generated if you have a disk problem or if you
   have too many files open.</BLOCKQUOTE
@@ -1188,4 +1188,4 @@
 ></DIV
 ></BODY
 ></HTML
->
\ No newline at end of file
+>
diff -Nur iptraf-2.7.0.orig/Documentation/rvnamed.8 iptraf-2.7.0/Documentation/rvnamed.8
--- iptraf-2.7.0.orig/Documentation/rvnamed.8	2000-03-13 13:03:59.000000000 +0100
+++ iptraf-2.7.0/Documentation/rvnamed.8	2003-06-02 23:01:26.000000000 +0200
@@ -11,7 +11,7 @@
 This program is only used by iptraf and, therefore, is useless alone.
 
 .SH FILES
- /var/local/iptraf/rvnamed.log - log file
+ /var/iptraf/rvnamed.log - log file
 
 .SH SEE ALSO
 README.rvnamed - documentation from the author
diff -Nur iptraf-2.7.0.orig/README.rvnamed iptraf-2.7.0/README.rvnamed
--- iptraf-2.7.0.orig/README.rvnamed	2001-11-16 02:30:21.000000000 +0100
+++ iptraf-2.7.0/README.rvnamed	2003-06-02 23:02:57.000000000 +0200
@@ -14,7 +14,7 @@
 menu, the IP Traffic Monitor will attempt to start rvnamed.  If for some 
 reason rvnamed is already running, IPTraf will use it immediately.  
 Otherwise, it will attempt to start rvnamed.  As of IPTraf 1.2.0, the
-rvnamed is placed together with the main IPTraf executable in /usr/local/bin.
+rvnamed is placed together with the main IPTraf executable in /usr/bin.
 
 When the traffic monitor is done, IPTraf tells rvnamed to quit.
 
diff -Nur iptraf-2.7.0.orig/src/dirs.h iptraf-2.7.0/src/dirs.h
--- iptraf-2.7.0.orig/src/dirs.h	2001-05-03 12:44:11.000000000 +0200
+++ iptraf-2.7.0/src/dirs.h	2003-06-02 23:00:39.000000000 +0200
@@ -26,7 +26,7 @@
  */
  
 #ifndef WORKDIR
-#define WORKDIR		"/var/local/iptraf"
+#define WORKDIR		"/var/iptraf"
 #endif
 
 #ifndef LOGDIR
@@ -34,7 +34,7 @@
 #endif
 
 #ifndef EXECDIR
-#define EXECDIR		"/usr/local/bin"
+#define EXECDIR		"/usr/bin"
 #endif
 
 /*
diff -Nur iptraf-2.7.0.orig/src/Makefile iptraf-2.7.0/src/Makefile
--- iptraf-2.7.0.orig/src/Makefile	2002-05-11 09:27:36.000000000 +0200
+++ iptraf-2.7.0/src/Makefile	2003-06-02 23:00:12.000000000 +0200
@@ -68,12 +68,12 @@
 # installation target directory.  The iptraf and rvnamed programs get
 # stored here.  iptraf also exec's rvnamed from this directory.
 
-TARGET		= /usr/local/bin
+TARGET		= /usr/bin
 
 # The IPTraf working directory; if you change this.  Starting with this
 # version, this definition overrides dirs.h.
 
-WORKDIR		= /var/local/iptraf
+WORKDIR		= /var/iptraf
 
 # The IPTraf log file directory.  IPTraf log files are placed here.
 # This definition overrides dirs.h
